src/project-1.js

```diff
@@ -3,137 +3,180 @@
 const multiplyByTen = (num) => {
   // return num after multiplying it by ten
   // code here
+  return num * 10;
```

Comment:
This is fine but you can also simplify the whole function to look like this:
```
const multiplyByTen = (num) => num * 10;
```
The value is automatically returned.